lib/ts/recipe/emailverification/types.ts

```diff
@@ -23,8 +23,9 @@ import EmailDeliveryIngredient from "../../ingredients/emaildelivery";
 import { GeneralErrorResponse } from "../../types";
 
 export type TypeInput = {
+    mode: "REQUIRED" | "OPTIONAL" | "OFF";
```

Comment:
Do we need the mode to be "OFF"? Cause if that's the case, then the user does not need to do EmailVerification.init at all.